tests/samples/expected/ocaml.ml

```diff
@@ -446,9 +446,11 @@ let topological_sort deps =
             else
               raise
               @@ Files_legacy.Files_error (ObjectFileNotFound (mk_mident node))
-      in node::List.fold_left (explore (node::path))
+      in
+      node::List.fold_left (explore (node::path))
         visited (List.map Files_legacy.get_file edges)
-  in List.rev @@ List.fold_left (fun visited (n, _) -> explore [] visited n) [] graph
+  in
+  List.rev @@ List.fold_left (fun visited (n, _) -> explore [] visited n) [] graph
```

Comment:
@aspiwack I'm fine with this PR if you're OK with this formatting (line break and no indenting).

Reply:
Yes, it's the most common style (there are many though. But this is the one I use, so it's also the best style).

Reply:
Oh no, it's not the one I'm using :cry: out of curiosity, what is your criterion for keeping the `in` at the end (as below in `let res = x + x in`) versus introducing a new line? Do you do that as soon as the let-binding spans more than one line?

Reply:
@yannham 

The design is that each form has two different shapes: single-line or multi-line. The selection between single-line and multi-line depends on the programmer's input: if I wrote something on several line, Topiary will use the multi-line shape, if I wrote it on one line, it will use the single-line shape.

The current style for `let` is

```ocaml
let x =
  …
in
```

and

```ocaml
let x = … in
```

The fact that

```ocaml
let x = foo
in
```

is considered single-line rather than multiline is a technical limitation due to details in the grammar.

Reply:
I see, this makes sense. Thanks!